string cmd
string target_mode
---
string success
